packages/frontend/src/components/common/PivotTable/index.tsx

```diff
@@ -347,6 +376,73 @@ const PivotTable: FC<PivotTableProps> = ({
                     </tr>
                 ))}
             </tbody>
+
+            {hasColumnTotals ? (
+                <tfoot>
+                    {data.columnTotals?.map((row, totalRowIndex) => (
+                        <tr key={`column-total-${totalRowIndex}`}>
+                            {/* shows empty cell if row numbers are visible */}
+                            {hideRowNumbers ? null : (
+                                <th
+                                    className={cellCx(
+                                        cellStyles.root,
+                                        cellStyles.rowNumber,
+                                    )}
+                                />
+                            )}
+
+                            {/* render the total label */}
+                            {data.columnTotalFields?.[totalRowIndex].map(
+                                (totalLabel, totalColIndex) =>
+                                    totalLabel ? (
+                                        <HeaderCell
+                                            key={`footer-total-${totalRowIndex}-${totalColIndex}`}
+                                            textAlign="right"
+                                            className={cellCx(
+                                                cellStyles.root,
+                                                cellStyles.header,
+                                            )}
+                                        >
+                                            {totalLabel.fieldId ?? 'Total'}
+                                        </HeaderCell>
+                                    ) : (
+                                        <th
+                                            key={`footer-total-${totalRowIndex}-${totalColIndex}`}
+                                            className={cellCx(
+                                                cellStyles.root,
+                                                cellStyles.rowNumber,
+                                            )}
+                                        />
+                                    ),
+                            )}
+
+                            {row.map((total, totalColIndex) => {
+                                const value = getColumnTotalValueFromAxis(
+                                    total,
+                                    totalColIndex,
+                                );
+
+                                return total ? (
+                                    <TotalCell
+                                        key={`column-total-${totalRowIndex}-${totalColIndex}`}
+                                        value={value}
+                                    >
+                                        {value.formatted}
```

Comment:
The total calculation doesn't have the same rules as before. We only show totals if:
- metric type is: number, count, count distinct, sum
- formatting is not 'percent'

This pivot table shows totals for any numeric column.